integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/apim/APIMLifecycleBaseTest.java

```diff
@@ -63,6 +63,8 @@
 
 public class APIMLifecycleBaseTest extends APIMWithMgwBaseTest {
     private static final Logger LOGGER = LoggerFactory.getLogger(APIMLifecycleBaseTest.class);
+    protected static final String SAMPLE_API_NAME = "PetStore";
+    protected static final String SAMPLE_API_CONTEXT = "petstore";
```

Comment:
Usually we don't keep API names like this in the base class

Reply:
issue is with the variable value or the class its placed in?

Reply:
Lets move the API names to the static section of the class

Reply:
Moved to testconstants f32cf4d